third_party/wizer/wizer.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Wizer interface for Wasm module to be initialized.
+ *
+ * This header provides several macros that allow a Wasm module written in C/C++
+ * to declare an initializer function, and ensure that global constructors (in
+ * C++'s case) are run at initialization time rather than on startup of the
+ * pre-initialized module.
+ */
+#ifndef _WIZER_H_
```

Comment:
Question: Do we need to include a copy of the Wizer license in the folder as well?
https://github.com/bytecodealliance/wizer/blob/main/LICENSE

Reply:
Done!